cli/main.go

```diff
@@ -143,7 +143,7 @@ func doSetup() int {
 		user.Password = readNewline(" > Password: ", stdin)
 		pwdHash, _ := bcrypt.GenerateFromPassword([]byte(user.Password), 11)
 
-		if _, err := db.Mysql.Exec("insert into user set name=?, username=?, email=?, password=?, created=UTC_TIMESTAMP()", user.Name, user.Username, user.Email, pwdHash); err != nil {
+		if _, err := db.Mysql.Exec("insert into user set name=?, username=?, email=?, password=?, admin=1, created=UTC_TIMESTAMP()", user.Name, user.Username, user.Email, pwdHash); err != nil {
```

Comment:
do we have to have admin as default? I would guess most people would prefer admin not to be the default user mode (especially in enterprise usage)

Reply:
I think, the first system user always should be an admin. That code executes only for initial setup of Semaphore.

Reply:
Agreed, sorry i misread this